packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/TableCardFormItems/TableCardFormContent.jsx

```diff
@@ -196,6 +196,28 @@ const TableCardFormContent = ({
     [onChange]
   );
 
+  const dataListItems = dataSection?.map((dataItem) => ({
```

Comment:
can we useMemo these for performance?  Change the ones above as well

Reply:
Memoized